shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -405,11 +405,14 @@ private String maybeGetInitialRouteFromIntent(Intent intent) {
     if (host.shouldHandleDeeplinking()) {
       Uri data = intent.getData();
       if (data != null && !data.getPath().isEmpty()) {
-        String pathAndQuery = data.getPath();
+        String pathAndQueryAndFragment = data.getPath();
```

Comment:
maybe just call it fullRouteName

Reply:
Thanks, I really hated that variable name.

I would name it fullRoute, or do you prefer fullRouteName?

Reply:
either is fine